implementations/rust/ockam/ockam_api/src/nodes/models/services.rs

```diff
@@ -365,7 +365,9 @@ impl StartOktaIdentityProviderRequest {
 #[rustfmt::skip]
 #[cbor(map)]
 pub struct ServiceStatus {
+	#[serde(rename = "address", serialize_with = "crate::util::serialize_as_multiaddr")]
```

Comment:
If we do this, then the `deserialize_with` attribute should be defined as well. Try to write a unit test to check that the serde methods work as expected.

Reply:
Would you prefer making the same approach as we did with inlets and outlets? Creating a version of the struct to handle the custom serialization without affecting api? I can also do the deserialization and tests. Whatever you think is better.

Reply:
I think you already solved this, so no custom deserialization or tests are needed.